tools/pythonpkg/tests/fast/arrow/test_arrow_list.py

```diff
@@ -0,0 +1,91 @@
+import duckdb
+import numpy as np
+try:
+    import pyarrow as pa
```

Comment:
Use `pytest.importorskip('pyarrow')` here instead please

Reply:
I've used this set up to conform to existing tests, looking at `importorskip` it would have to be called in each test that requires the module, no?

Or maybe there is some way through inheritance to make this more easy to use but I feel like this issue is out of scope of this fix and should be collectively tackled with all the other tests that currently use this set up.

I do agree that `importorskip` looks like a cleaner and more integrated way to get this behavior than the current solution.

Reply:
@Tishj it can be used at the module level as well, and is pretty vastly preferable